app/Middleware/Global/AccessAuth.ts

```diff
@@ -20,26 +20,21 @@ export default class AccessAuth {
         required: 'You must provide an access_id header to access the api. Read more on ...',
```

Comment:
If the id of the access model is not used, we should use the access_id as a primary key and remove the id column

Reply:
Yes good idea! What we could do is to have the id as uuid and remove the access_id but still require it as is for any API request. Wdyt?